source/frontend/src/features/mapSideBar/property/tabs/crown/CrownDetailsTabView.tsx

```diff
@@ -0,0 +1,67 @@
+import { Feature, Geometry } from 'geojson';
+
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { StyledSummarySection } from '@/components/common/Section/SectionStyles';
+import { TANTALIS_CrownLandTenures_Feature_Properties } from '@/models/layers/crownLand';
+import { exists, prettyFormatDate } from '@/utils';
+
+export interface ICrownDetailsTabViewProps {
+  crownFeature?: Feature<Geometry, TANTALIS_CrownLandTenures_Feature_Properties> | undefined;
+}
+
+export const CrownDetailsTabView: React.FunctionComponent<ICrownDetailsTabViewProps> = ({
+  crownFeature,
+}) => {
+  if (!exists(crownFeature)) {
+    return null;
+  } else {
+    return (
+      <StyledSummarySection>
+        <Section header="Crown Details">
+          <SectionField label="Tenure stage">
+            {crownFeature?.properties?.TENURE_STAGE ?? ''}
+          </SectionField>
+          <SectionField label="Tenure status">
+            {crownFeature?.properties?.TENURE_STATUS ?? ''}
+          </SectionField>
+          <SectionField label="Tenure type">
+            {crownFeature?.properties?.TENURE_TYPE ?? ''}
+          </SectionField>
+          <SectionField label="Tenure sub type">
+            {crownFeature?.properties?.TENURE_SUBTYPE ?? ''}
+          </SectionField>
+          <SectionField label="Tenure purpose">
+            {crownFeature?.properties?.TENURE_PURPOSE ?? ''}
+          </SectionField>
+          <SectionField label="Crown lands file">
+            {crownFeature?.properties?.CROWN_LANDS_FILE ?? ''}
+          </SectionField>
+          <SectionField label="Tenure document">
+            {crownFeature?.properties?.TENURE_DOCUMENT ?? ''}
+          </SectionField>
+          <SectionField label="Tenure expiry">
+            {prettyFormatDate(crownFeature?.properties?.TENURE_EXPIRY)}
```

Comment:
This appears to be changing the date based on the assumption that the date occurs at 00:00:00 UTC - I'm guessing we want to keep the date as is: 

Ie: "2025-08-22Z" from the layer is displayed as Aug 21, 2025